src/index.js

```diff
@@ -3,13 +3,23 @@ import ReactDOM from "react-dom";
 import "./index.css";
 import App from "./App";
 import { makeServer } from "./server";
+// import reportWebVitals from './reportWebVitals';
+import {BrowserRouter, Routes, Route } from 'react-router-dom'
 
 // Call make Server
 makeServer();
 
 ReactDOM.render(
   <React.StrictMode>
-    <App />
+    <BrowserRouter>
```

Comment:
If you would like to keep index.js clean, make a router folder in pages folder, and then import that to index.js. This is also fine though